service-base/src/main/java/org/eclipse/hono/service/registration/RegistrationService.java

```diff
@@ -22,53 +22,14 @@
 
 /**
  * A service for keeping record of device identities.
- * 
+ *
```

Comment:
This description should probably be modified to reflect the fact, that the interface only contains the mandatory operations (if that is what you have in mind) ...